### PR TITLE
Add domain support for confidentials

### PR DESCRIPTION
**Pull request by philogicae**
This is a draft, I just wanted to test some stuff and I ended up doing few fixes/improvements:

- Continued confidential current implementation (needed for the following changes)
- Fixed /settings/domain/[domain] for cocos
- Updated /settings/domain/new to add coco support
- Display the right entity type for cocos

### CODE REVIEW

**Comment by hoh** [comment]
Is this still a draft as described in the body of the PR ?

**Comment by philogicae** [comment]
> Is this still a draft as described in the body of the PR ?

1) Since the coco instance implementation on front isn't done yet, @amalcaraz / @gmolki have to check if my changes here are fine

2) I still have to check with @aliel why DNS resolution is currently not working for coco domains, despite the changes

File: src/components/pages/dashboard/ManageDomain/cmp.tsx [hunk 1]
```diff
@@ -42,6 +42,7 @@ export default function ManageDomain() {
 
   const name = domain.name || ellipseAddress(domain.id)
   const typeName = EntityTypeName[domain.type]
+  const is_confidential = domain.target == EntityDomainType.Confidential
```
Comment (amalcaraz) [inline src/components/pages/dashboard/ManageDomain/cmp.tsx:45]: in javascript by convention variables are camelCase, please replace it everywhere it has been set in snake_case

**Review by amalcaraz — CHANGES_REQUESTED** [review]
LGTM, minor change requested tho

**Pull request merged**